src/components/admin/ActionTable.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import Toolbar from '@material-ui/core/Toolbar';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import TableFooter from '@material-ui/core/TableFooter';
+import TablePagination from '@material-ui/core/TablePagination';
+
+import styles from './admin.module.css';
+import TablePaginationActions from './TablePaginationActions';
+
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+import {Typography} from '@material-ui/core';
+
+function createData(id, date, message) {
+  return {id, date, message};
+}
+
+const rows = [
+  createData('1', new Date(),
+      '[NOOGLER CHECK 1]: A partner of a noogler requires access'),
+  createData('2', new Date(),
+      '[DATABASE]: A new member could not be added to the database.'),
+  createData('3', new Date(),
+      '[NOOGLER CHECK 2]: A partner of a noogler requires access'),
+];
```

Comment:
Could be done with JS magic, if you want:
```suggestion
const rows = Array.from([
  '[NOOGLER CHECK 1]: A partner of a noogler requires access',
  '[DATABASE]: A new member could not be added to the database.',
  '[NOOGLER CHECK 2]: A partner of a noogler requires access'
  ]), (message, id) => ({
    id,
    message,
    date: new Date()
  }));
```